app/client/cli/debug.go

```diff
@@ -0,0 +1,144 @@
+package cli
```

Comment:
Not actionable: Big fan of how this is in a separate file now.

We can do something similar for `infra` soon. CC @okdas @vlad-she